ironfish-rust/src/keys/public_address.rs

```diff
@@ -19,19 +19,14 @@ pub const PUBLIC_ADDRESS_SIZE: usize = 32;
 /// transmission key. Using the incoming_viewing_key allows
 /// the creation of a unique public addresses without revealing the viewing key.
 #[derive(Clone, Copy)]
-pub struct PublicAddress {
-    /// The transmission key is the result of combining the diversifier with the
-    /// incoming viewing key (a non-reversible operation). Together, the two
-    /// form a public address to which payments can be sent.
-    pub(crate) transmission_key: SubgroupPoint,
-}
+pub struct PublicAddress(pub(crate) SubgroupPoint);
 
 impl PublicAddress {
     /// Initialize a public address from its 32 byte representation.
     pub fn new(address_bytes: &[u8; PUBLIC_ADDRESS_SIZE]) -> Result<PublicAddress, IronfishError> {
-        let transmission_key = PublicAddress::load_transmission_key(&address_bytes[0..])?;
+        let public_address = PublicAddress::load_public_address(&address_bytes[0..])?;
```

Comment:
Bit of a nit but what do you think about just making `new` do the stuff in `load_public_address` and removing the `load_public_address` function? There used to be more stuff here but now it feels unnecessary to keep that extra layer of indirection